applications/StructuralMechanicsApplication/custom_utilities/shell_cross_section.cpp

```diff
@@ -15,1258 +15,1473 @@
 #define M_PI 3.1415926535897932384626433832795
 #endif // M_PI
 
-

```

Comment:
This wasn't changed by us, it was already there.
But we can change it to the Kratos-global PI